system-parachains/constants/src/genesis_presets.rs

```diff
@@ -66,3 +67,20 @@ where
 
 /// The default XCM version to set in genesis config.
 pub const SAFE_XCM_VERSION: u32 = xcm::prelude::XCM_VERSION;
+
+pub fn remove_phantom_fields(value: &mut Value) {
```

Comment:
@michalkucharczyk wdyt here?

Reply:
`[#serde(skip)]` shall be used for phantom fields.

Reply:
This means i have to open a PR in polkadot-sdk to include the` [#serde(skip)]` annotation for these pallets because they don't actually have them example is the `pallet-bridge-messages ` pallet? @michalkucharczyk  @bkontur

Reply:
Yes, PR shall be opened. In that case, IMO we could keep this function until clean-up PR is merged (some annotations would be nice).

As alternative (and probably better solution) you could experiment with removing the fields that were not customized from the final JSON (see [this discussion](https://github.com/paritytech/polkadot-sdk/pull/5327#issuecomment-2343838522)).

Reply:
> This means i have to open a PR in polkadot-sdk to include the` [#serde(skip)]` annotation for these pallets because they don't actually have them example is the `pallet-bridge-messages ` pallet? @michalkucharczyk @bkontur

@dharjeezy actually, this pallet was fixed already in the master: https://github.com/paritytech/polkadot-sdk/blob/master/bridges/modules/messages/src/lib.rs#L539-L540, but if you have there any others, please feel free to open PR to polkadot-sdk